### PR TITLE
feat: create atomic roles for item class

### PR DESCRIPTION
**Pull request by voknil**
https://oat-sa.atlassian.net/browse/ADF-154

- Created atomic roles for item classes (ItemClassNavigator, ItemClassEditor, ItemClassCreator) which include base BackOffice for being able to login
- Migration and install script
- Non-breaking fix in RdfController for proper division of access to view and edit methods In child controllers

How to test:

- Create users with atomic roles
- Log in and check that only specified in the task abilities are matched

Dependent PRs:

- [core](https://github.com/oat-sa/tao-core/pull/2852)
- [generis](https://github.com/oat-sa/generis/pull/896)

### CODE REVIEW

File: actions/class.Items.php [hunk 1]
```diff
@@ -153,6 +153,15 @@ public function getOntologyData()
         return parent::getOntologyData();
     }
 
+    /**
+     * view class label action
+     * @requiresRight classUri READ
+     */
+    public function viewClassLabel()
```
Comment (shpran) [inline actions/class.Items.php:160]: Can we declare here a return type?

Comment (voknil) [reply actions/class.Items.php:160]: parent method doesn't has a return type, so I think this one shouldn't either, or we can not return at all, but all other methods are written in the same way

**Comment by gabrielfs7** [comment]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

File: actions/class.Items.php [hunk 2]
```diff
@@ -153,6 +153,15 @@ public function getOntologyData()
         return parent::getOntologyData();
     }
 
+    /**
+     * view class label action
```
Comment (gabrielfs7) [inline actions/class.Items.php:157]: Please, remove unnecessary comments
```suggestion
```

File: actions/class.Items.php (hunk 1 above)
Comment (gabrielfs7) [inline actions/class.Items.php:160]: I think we do not need this here, instead we should use annotations and create proper method: 

https://github.com/oat-sa/tao-core/pull/2852/files#r612168872

**Pull request closed without merging**